content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -103,7 +103,8 @@ different Kubernetes components.
 | `DisableCloudProviders` | `false` | Alpha | 1.22 | |
 | `DisableKubeletCloudCredentialProviders` | `false` | Alpha | 1.23 | |
 | `DownwardAPIHugePages` | `false` | Alpha | 1.20 | 1.20 |
-| `DownwardAPIHugePages` | `false` | Beta | 1.21 | |
+| `DownwardAPIHugePages` | `false` | Beta | 1.21 | 1.22 |
```

Comment:
```suggestion
| `DownwardAPIHugePages` | `false` | Beta | 1.21 | 1.21 |
```

Reply:
Oh so that's half inclusive to the left. Updated.